db/sqlinterfaces.c

```diff
@@ -5683,6 +5683,7 @@ void cleanup_clnt(struct sqlclntstate *clnt)
         }
         free(clnt->context);
         clnt->context = NULL;
+        clnt->ncontext = 0;
```

Comment:
I assume this is for completeness? We did not get a crash/fail because of context not set to 0, did we?

Reply:
Yes- for completeness.  I didn't see it crash because this line was missing.